src/libshadertrap/include/libshadertrap/parser.h

```diff
@@ -23,6 +23,7 @@
 #include <string>
 #include <utility>
 #include <vector>
+#include <unordered_set>
```

Comment:
Includes should be in alphabetical order. But we will shortly get to using our various checking tools to help identify these kinds of things.